src/status_im/ui/screens/wallet/assets/views.cljs

```diff
@@ -0,0 +1,84 @@
+(ns status-im.ui.screens.wallet.assets.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.status-bar :as status-bar]
+            [status-im.ui.components.toolbar.view :as toolbar]
+            [status-im.ui.components.button.view :as button]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.styles :as component.styles]
+            [status-im.ui.components.button.styles :as button.styles]
+            [status-im.ui.screens.wallet.components.views :as components]
+            [status-im.ui.screens.wallet.assets.styles :as assets.styles]
+            [status-im.ui.screens.wallet.main.styles :as main.styles]
+            [status-im.ui.components.tabs.views :as tabs]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.components.styles :as components.styles]))
+
+
+(defview my-token-tab-title [active?]
+  (letsubs [ {:keys [token-symbol]} [:token-balance]]
+    [react/text {:uppercase? true
+                 :style      (assets.styles/tab-title active?)}
+     (i18n/label :t/wallet-my-token {:symbol token-symbol})]))
+
+(defview my-token-tab-content []
+  (letsubs [syncing? [:syncing?]
+            {:keys [token-symbol
+                    token-value
+                    usd-value]} [:token-balance]]
+    [react/view components.styles/flex
+     [react/view {:style assets.styles/total-balance-container}
```

Comment:
For ease of reading we might want to extract some of those as standalone components?